### PR TITLE
[PHP] Add PHP 7.4 Type Property Rector [ref #638]

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Good idea :+1: 

What do you mean by "and default invalid values"?
That seems like task for PHPStan

**Comment by carusogabriel** [comment]
@TomasVotruba Something like 
```php
/** @var bool */
public $foo = 1;
```

We can't refactor it, even if
> That seems like task for PHPStan

**Comment by carusogabriel** [comment]
@TomasVotruba A tip: be clear in your commits message. For example: e5af6ee its message doesn't say you are adding a test case or something. For future debug and `git biset`, having those messages clear can save us some time :smile:

**Comment by TomasVotruba** [comment]
Such simple check can be done :)

That's a fixup commit. They'll be revised to one, see:
https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

File: packages/Php/tests/Rector/TypedPropertyRector/Wrong/ClassWithProperty.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * @var void
+     */
+    private $shouldBeSkipped;
+
+    /**
+     * @var callable
+     */
+    private $shouldBeSkippedToo;
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/Php/tests/Rector/TypedPropertyRector/Wrong/ClassWithProperty.php]: Good idea, added: https://github.com/rectorphp/rector/pull/643/files#diff-ce72093f292ee192f2228cc9d1fc5ba1R40

**Pull request merged**